.github/workflows/npmjs-cd.yml

```
@@ -0,0 +1,74 @@
+jobs:
+  updates_version:
+    name: Updates package version
+    runs-on: ubuntu-latest
+
+    permissions:
+        contents: write
+        packages: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)